### PR TITLE
Simplified_Chinese_fonts_Support

### PR DESCRIPTION
**Pull request by Kingwayer**

### CODE REVIEW

File: alpine/5/Dockerfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ ARG INVOICENINJA_VERSION
 ARG REPOSITORY=invoiceninja/invoiceninja
 ARG FILENAME=react-invoiceninja.tar
 
-RUN set -eux; apk add curl unzip grep
+RUN set -eux; apk add curl unzip grep vim nano
```
Comment (turbo124) [inline alpine/5/Dockerfile:13]: vim/nano not required for production, you may need these whilst debugging, but they should be removed for the production builds

File: alpine/5/Dockerfile [hunk 2]
```diff
@@ -38,14 +38,20 @@ RUN mv /usr/local/etc/php/php.ini-production /usr/local/etc/php/php.ini
...
 RUN set -eux; \
     apk add --no-cache \
     font-isas-misc \
     supervisor \
     mysql-client \
     chromium \
+    tzdata \
     ttf-freefont \
+    font-noto-cjk \
+    glibc-i18n \
```
Comment (turbo124) [inline alpine/5/Dockerfile:54]: can you explain the need for txdata + glibc ?

File: alpine/5/Dockerfile (hunk 1 above)
Comment (Kingwayer) [reply alpine/5/Dockerfile:13]: no problem to remove these unnecessary cmd.

File: alpine/5/Dockerfile (hunk 2 above)
Comment (Kingwayer) [reply alpine/5/Dockerfile:54]: tzdata is for timing accurate, glibc is for  i18n language support, you can remove tzdata but I believe better keep it.

see explanation from chatgpt below:

````
tzdata is a package that provides the time zone database required by various software for time zone and daylight saving time calculations. It contains data representing the history of local time for many locations around the globe.

Key Functions of tzdata
Time Zone Database:

tzdata contains the comprehensive database of time zone information. This includes historical, current, and future time zone and daylight saving time (DST) information for many locations worldwide.
Standardized Time Calculations:

It enables consistent and accurate time zone conversions and calculations across different systems and applications. This is essential for tasks like scheduling events, logging timestamps, and other time-related operations.
Support for Various Applications:

Many software systems and programming languages rely on tzdata for correct time zone handling. For instance, databases, web servers, and various application frameworks use this data to ensure their time-related functionality is accurate.
````



```
glibc-i18n is a component of the GNU C Library (glibc) that focuses on internationalization (i18n) and localization (l10n) support. The GNU C Library is a widely-used standard C library that provides many fundamental system functions and services, and glibc-i18n specifically handles tasks related to making applications suitable for various languages and cultural conventions.

Key Features
Character Set Support:

glibc-i18n supports various character encodings, such as UTF-8, ISO-8859-1, and others, ensuring that applications can handle different character sets.
Localization:

It allows customization of application behavior based on locale settings, such as date formats, currency symbols, and number formats.
Message Translation:

Supports the GNU gettext framework, which enables applications to display messages in different languages based on user preferences.```

**Comment by turbo124** [comment]
thanks for this, i tried building the container, but it fails. 

have you tested the build?

```
ERROR: failed to solve: process "/bin/sh -c set -eux;     apk add --no-cache     font-isas-misc     supervisor     mysql-client     chromium     tzdata     ttf-freefont     font-noto-cjk     glibc-i18n     ttf-dejavu" did not complete successfully: exit code: 1
```

build command

```
docker build --build-arg="INVOICENINJA_VERSION=5.9.4" alpine/5/ -t ghpr588 --progress=plain
```

**Comment by Kingwayer** [comment]
Yes, I have build it and running well from one of my VM which hosted in my PVE from my home ( I am using Cloudflare tunnel connect it from outside), I was remember last time when I ask friend to submit PR the first time is fails, He says you docker build cmd is need improve.

https://home.kingwayer.com and https://invoice.kingwayer.com

image

**Pull request closed without merging**